### DISCUSSION TITLE
图片无法正常显示

### DISCUSSION THREAD

**Discussion by Oaciqihz** [RandyPen/sui-move-intro-course-zh#5] [Q&A]
![image](https://github.com/RandyPen/sui-move-intro-course-zh/assets/108737430/1d0dcdc6-59ba-44d0-bdef-ca8b2b226b4b)
或许得换相对路径？

![image](https://github.com/RandyPen/sui-move-intro-course-zh/assets/108737430/83e4519f-1379-425a-a3b5-835a57a660cc)
跳转路径错误403，这里好像得拼github地址

另外后面方便开通issues功能吗？有问题可以直接提issues。

**Comment by RandyPen** [comment]
@uvd 咋整合适
